confluence-slack-server-integration-plugin/src/test/java/it/com/atlassian/confluence/plugins/slack/util/SlackWebTestBase.java

```diff
@@ -24,7 +35,41 @@
     @RegisterExtension
     static RuleAdaptorExtension webDriverScreenshotRule = new RuleAdaptorExtension(new WebDriverScreenshotRule());
 
-    protected void closeAuiFlags() {
-        confluence.get().getTester().getDriver().executeScript("AJS.$('.aui-flag').each(function(it) { this.close(); } );");
+    protected void closeAuiFlags(final long sleepMsBeforeChecking) {
```

Comment:
Just curious:
1. Did you implement this method or did you get it from somewhere?
2. In your tests, did Jira still show random flags everywhere as in previous versions?
3. Did you check if in the new Jira version there's a better way of getting rid of flags that actually works? Like a property or dark feature?

Reply:
1. I implemented this method on my own as a result of a lot of iterations. Maybe not all actions here are required to close flags. It may be investigated and addressed later if needed.
2. Yes, the same test (`ConfigurationWebTest.deleteConfiguration`, see [one of the latest runs](https://github.com/atlassian-labs/atlassian-slack-integration-server/actions/runs/2041390379)) failed over and over again and I saw AUI flags on screenshots. After I managed to close them, the test started passing without issues. BTW it is related to Confluence tests, not Jira ones. They were indirectly affected by the change of Selenium dependencies versions used in Jira plugin.
3. I tried searching on the web, but found nothing useful.